<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../Resources/icon.png</normaloff>../Resources/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="RecordName">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>521</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>הכנס את שם ההקלטה</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="RecordText">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>190</y>
      <width>521</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>הכנס את הטקסט שיהיה בהקלטה</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>201</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>הוספת הקלטה</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>240</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>MV Boli</family>
     </font>
    </property>
    <property name="text">
     <string>בחר תמונה להקלטה</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AddRecorButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>310</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>הוסף הקלטה</string>
    </property>
   </widget>
   <widget class="QLabel" name="ErrorMsgLabel">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>360</y>
      <width>521</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>שגיאה</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
